snippets/csharp/VS_Snippets_CLR_System/system.datetime.compare/cs/compare1.cs

```diff
@@ -18,8 +18,9 @@ public static void Main()
          relationship = "is later than";
 
       Console.WriteLine("{0} {1} {2}", date1, relationship, date2);
+
+      // The example displays the following output:
```

Comment:
The output is slightly different on try.net because of culture differences. 

```suggestion
      // The example displays the following output for en-us culture:
```